src/TraceListener/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 [assembly: Guid("3aada9bf-fc15-42fd-ace5-4cb2f11c1f2a")]
 
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.TraceListener.Net45.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.TraceListener.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
```

Comment:
maybe have `netcoreapp10` in the name of test assembly?

Reply:
Done